src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -20,6 +20,8 @@
      */
     UniqueTagList getTags();
 
+    int getSequenceNumber();
```

Comment:
following Interface Segregation Principle, we should be clear whether or not a method should or should not be placed in an interface. In this case, getSequenceNumber() may not be. But this is arguable.